libraries/provider_database_mysql_user.rb

```diff
@@ -105,7 +105,7 @@ def ishash?
             test_sql += " AND Db='#{new_resource.database_name}'" if new_resource.database_name
             test_sql_results = test_client.query test_sql
 
-            incorrect_privs = true if test_sql_results.empty?
+            incorrect_privs = true if test_sql_results.find { |k| k['User'] }.nil?
```

Comment:
this is unrelated to the original pull request, but it fixes the bug in the mysql provider which causes the travis build to fail, i figured i might as well fix this while i'm at it. so the travis build is successful now.